python/GafferUI/ColorChooserPlugValueWidget.py

```diff
@@ -103,3 +171,49 @@ def __allComponentsEditable( self ) :
 					return False
 
 		return True
+
+def saveDefaultOptions( colorChooser, keyPrefix, scriptPath = None ) :
+
+	for k in [ "visibleComponents", "staticComponent", "colorFieldVisible" ] :
```

Comment:
Just a passing thought, but any preference to looping over the plug types here rather than duplicating the metadata deregister/register calls?

Reply:
Yes, that's a bit cleaner now, in https://github.com/GafferHQ/gaffer/pull/6003/commits/c09ac5960ab3f2762c46b33f31a9611448ce0362